components/rmrk/Gallery/GalleryItem.vue

```diff
@@ -30,6 +30,8 @@
             'is-12 is-theatre': viewMode === 'theatre',
             'is-6 is-offset-3': viewMode === 'default',
           }"
+          @mouseover="showNavigation = true"
```

Comment:
Hey @kkukelka you are using here two unrelated events. Namely, The mouseenter and mouseover events are triggered when you move the mouse over an element.
mouseenter only triggers when the mouse enters the element on which it is set. The counterpart event is mouseleave.
mouseover triggers when the mouse enters the element or any of its children. Its counterpart is mouseout.

Reply:
thanks for the hint, I always mix those up 😅